queries.sql

```diff
@@ -198,4 +198,15 @@ Use EXPLAIN ANALYZE to check what is happening.
 */
 
 CREATE INDEX visits_animals_id_asc ON visits(animals_id ASC);
-explain analyze SELECT COUNT(*) FROM visits where animals_id = 4;
\ No newline at end of file
+explain analyze SELECT COUNT(*) FROM visits where animals_id = 4;
+
+-----------------------------------------------
+/*
+Find a way to improve execution time of the other two queries.
+Use EXPLAIN ANALYZE to check what is happening.
+*/
+CREATE INDEX visits_vets_id_asc ON visits(vets_id DESC);
+explain analyze SELECT * FROM visits where vets_id = 2;
+
+CREATE INDEX owners_email_asc ON owners(email ASC);
+explain analyze SELECT * FROM owners where email = 'owner_18327@mail.com';
```

Comment:
- [ ] Good job so far 👍 but all the indexes should be placed in the schema.sql file. Kindly rectify that 💯 
- [ ] (OPTIONAL) It would be great if you could improve the performance of the third query as right now its time is still too big.